### ISSUE TITLE
Why pm2?

### ISSUE THREAD

**Issue by SantoshSrinivas79** [andruschka/pm2-meteor#3]
I am sure this is obvious to many, but could you put in something in the description as to why works better in what situations?

**Comment by andruschka** [comment]
mup uses forever - and PM2 is the more functional counterpart to forever. 
We tested it on freebsd and ubuntu with some of our Meteor apps and it worked great! (You can for example run 4 instances of your app in cluster__mode without the need for changing your code base)

**Comment by andruschka** [comment]
TLDR: We think PM2 is currently the best and simplest process manager.

**Comment by andruschka** [comment]
I will add an info in the Readme later - thanks!

**Comment by mikeislearning** [comment]
I'd be interested in hearing more details about this as well. So far I've been pretty happy with mup, but I'm always open to other options!

**Comment by andruschka** [comment]
First we also used mup, but experienced some issues:
1. you cant deploy from a git repo
2. mup uses forever - and forever had unaccountable crashes on our servers 
(we were not alone with this and heared that a lot of people switched to pm2, so we tried it out  - and it works great and has some cool features like load balancing..)
3. mup wont work on freebsd
4. we wanted to set the deployment directory ourself
(maybe you have a nodejs deployment user and want the apps in the homedir ?!)
5. we wanted to pass a settings file path instead the whole settings to the deployment-tool

This is why we build our own "mup" - which - for us- solves the problems mentioned above.
